### ISSUE TITLE
bug(mat-chip-grid): Accessibility Issue: ARIA Attribute Unsupported or Prohibited

### ISSUE THREAD

**Issue by michaelstokes93** [angular/components#30152]
### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

The `mat-chip-grid` component in Angular Material includes an `aria-disabled` and `aria-invalid` attribute by default, even when not explicitly set in the application. This attributes are not valid on `mat-chip-grid` elements and results in accessibility violations flagged by tools like Siteimprove.

### Reproduction

1. Use a `mat-chip-grid` in an Angular component.
2. Render the component and inspect the DOM.
3. Observe that `aria-disabled="false"` and `aria-false="false"`is automatically added to the `mat-chip-grid`.

### Expected Behavior

The `aria-disabled` and `aria-invalid` attributes should only be applied to elements or roles that support it, such as interactive components.

### Actual Behavior

The `aria-disabled` and `aria-invalid` attributes is added to the `mat-chip-grid` component, which results in accessibility violations.

### Environment

- Angular: 16.2.0
- CDK/Material: 16.2.0
- Browser(s): Chrome
- Operating System (e.g. Windows, macOS, Ubuntu): Windows

**Comment by crisbeto** [comment]
According to the standard for `role="grid"`, `aria-disabled` and `aria-invalid` are allowed: https://www.w3.org/TR/wai-aria-1.0/roles#grid
